src/constants/localStorage.ts

```diff
@@ -10,6 +10,7 @@ export const storageKeys = {
   ANCHOR_SESSION_PARAMS: 'ANCHOR_SESSION_PARAMS',
   STELLAR_OPERATIONS: 'STELLAR_OPERATIONS',
   TOKEN_BRIDGED_AMOUNT: 'TOKEN_BRIDGED_AMOUNT',
+  SIWE_SIGNATURE_KEY_PREFIX: 'siwe-signature-',
```

Comment:
Is the capitalization format required like this? If it's not, I'd say we make it similar to the others and capitalize to `SIWE_SIGNATURE_`